custom_components/elmo_iess_alarm/binary_sensor.py

```diff
@@ -110,7 +133,7 @@ def __init__(
         self._unique_id = unique_id
         self._sensor_id = sensor_id
         self._sensor_type = sensor_type
-        self._name = f"{DOMAIN} {name}"
```

Comment:
I think it's better if we include the `DOMAIN` as a prefix. This prevents any possible clash with other integrations. If you agree, you can rollback this change!

Reply:
Now that I see your code, I would say this change is correct, but in the new `generate_entity_name(config)` (assuming you agree with the suggestion) you should include the `DOMAIN` part.

Let me know what you think!